hypergan/util/ops.py

```diff
@@ -261,13 +263,16 @@ def lrelu_sq(x):
     dim = len(x.get_shape()) - 1
     return tf.concat(axis=dim, values=[lrelu(x), tf.minimum(tf.abs(x), tf.square(x))])
 
-def linear(input_, output_size, scope=None, mean=0., stddev=0.02, bias_start=0.0, with_w=False, regularizer=None):
+def linear(input_, output_size, scope=None, mean=0., stddev=0.02, bias_start=0.0, with_w=False, regularizer=None, gain=1.0):
     shape = input_.get_shape().as_list()
 
+    initializer = tf.orthogonal_initializer(1.0)
```

Comment:
Initialization seems like it might be a hyperparam.

This might be related to https://www.pivotaltracker.com/story/show/142115353